Documentation/ceph-cluster-crd.md

```diff
@@ -1319,128 +1319,63 @@ The features available from the external cluster will vary depending on the vers
 ```
 
-#### CephCluster example (consumer)
+>Note: Use format bash, as to keep up with the further steps, and rbd-data-pool and cephfs-filesystem, should be created in advance.
 
-Assuming the above section has successfully completed, here is a CR example:
+2. After that you need to copy the bash output.
 
-```yaml
-apiVersion: ceph.rook.io/v1
-kind: CephCluster
-metadata:
-  name: rook-ceph-external
-  namespace: rook-ceph-external
-spec:
-  external:
-    enable: true
-  crashCollector:
-    disable: true
-  # optionally, the ceph-mgr IP address can be pass to gather metric from the prometheus exporter
-  #monitoring:
-    #enabled: true
-    #rulesNamespace: rook-ceph
-    #externalMgrEndpoints:
-      #- ip: 192.168.39.182
-    #externalMgrPrometheusPort: 9283
-```
+#### Run commands on machine that has access to your Kubernetes cluster:
 
-Choose the namespace carefully, if you have an existing cluster managed by Rook, you have likely already injected `common.yaml`.
-Additionally, you now need to inject `common-external.yaml` too.
+1. Deploy Rook-Ceph, create [common.yaml](/deploy/examples/common.yaml), [crds.yaml](/deploy/examples/crds.yaml) and [operator.yaml](/deploy/examples/operator.yaml) manifests.
 
-You can now create it like this:
+2. Just paste the above output from `create-external-cluster-resources.py` into your current shell, it will export all the data.
 
+3. Run the [import](/deploy/examples/import-external-cluster.sh) script.
 ```console
-kubectl create -f deploy/examples/cluster-external.yaml
+. import-external-cluster.sh
 ```
 
-If the previous section has not been completed, the Rook Operator will still acknowledge the CR creation but will wait forever to receive connection information.
-
-> **WARNING**: If no cluster is managed by the current Rook Operator, you need to inject `common.yaml`, then modify `cluster-external.yaml` and specify `rook-ceph` as `namespace`.
-
-If this is successful you will see the CepCluster status as connected.
+4. Create [common-external.yaml](/deploy/examples/common-external.yaml) and [cluster-external.yaml](/deploy/examples/cluster-external.yaml)
 
+5. External-cluster will be connected to standalone ceph cluster, verify using:
 ```console
-kubectl get CephCluster -n rook-ceph-external
+kubectl -n rook-ceph-external  get CephCluster
 ```
-
 >```
 >NAME                 DATADIRHOSTPATH   MONCOUNT   AGE    STATE       HEALTH
 >rook-ceph-external   /var/lib/rook                162m   Connected   HEALTH_OK
 >```
 
+>Note: 
+>1) If there is already a cluster managed by Rook in 'rook-ceph' then create [cluster-external-management.yaml](/deploy/examples/cluster-external-management.yaml)
+>
+>2) If you plan to create CRs (pool, rgw, mds, nfs) in the external cluster, you **MUST** inject the client.admin keyring as well as injecting `cluster-external-management.yaml`
+>
+>3) If the admin key is to be used by the external cluster, set the following variable:
```

Comment:
Should we keep this point??

Reply:
I vote to remove discussion of the admin key.

Reply:
+1